src/uwtools/cli.py

```diff
@@ -852,22 +852,26 @@ def _add_subparser(subparsers: Subparsers, name: str, helpmsg: str) -> Parser:
 def _add_subparser_for_driver(
     name: str,
     subparsers: Subparsers,
+    with_batch: Optional[bool] = False,
```

Comment:
I made some changes in this module to support executing an assets-only driver (i.e. one that does not execute a component, but just provisions content to a run directory) like `cdeps` via the command line. Our other two existing assets-only drivers,`schism` and `ww3`, are slated for use only in a coupled driver and are not connected to the CLI, so this is the first time the need has arisen.